hack/e2e-test.sh

```diff
@@ -55,6 +55,11 @@ function startup {
 function kind_load {
     if [ $CREATE_KIND_CLUSTER == 'true' ]
     then
+        export E2E_IMAGES=$(grep '.Image(' ./test/e2e/e2e_test.go | cut -d '"' -f2 | sort | uniq)
+        while read -r E2E_IMAGES; do
+            docker pull $E2E_IMAGES
+            $KIND load docker-image $E2E_IMAGES --name $KIND_CLUSTER_NAME
+        done <<< "$E2E_IMAGES"
```

Comment:
For me, parsing the .go file is a bit too much, we currently have only one additional image. Also, I don't think that going forward there is any guarantee that  `e2e_test.go` will remain the only file of interest or that the potentially new images will also be provided as string literals and not const or var.

Reply:
@trasc I understand that parsing the go file to pick the image names is not ideal but there doesn't seem to be any other source for these image names.
Should we add a variable in `e2e-test.sh` in where people can manually add the image names that need to be pre-pulled?
Do you have any recommendation?

Reply:
We probably will only use one image that gives us good control over the behavior (how long it runs, which exit code it returns). Putting it in a variable should be enough.